server/src/controllers/Events/resolver.ts

```diff
@@ -358,6 +363,25 @@ export class EventResolver {
     });
   }
 
+  @Query(() => [EventWithVenue])
+  async dashboardEvents(
+    @Ctx() ctx: Required<ResolverCtx>,
+    @Arg('limit', () => Int, { nullable: true }) limit?: number,
+    @Arg('showAll', { nullable: true }) showAll?: boolean,
```

Comment:
```suggestion
    @Arg('showAll', { nullable: true }) showAll?: boolean,
```
Do you have plans for this?

Reply:
Right, not really.